lang/en_US.json

```diff
@@ -308,6 +308,51 @@
     "Latest": "Latest",
     "Less": "Less",
     "Both": "Both",
+    "Download Emulator": "Download Emulator",
+    "Global Points Ranking": "Global Points Ranking",
+    "Global Beaten Games Ranking": "Global Beaten Games Ranking",
+    "Join us on Discord": "Join us on Discord",
+    "Become a Patron": "Become a Patron",
+    "Documentation": "Documentation",
+    "FAQ": "FAQ",
+    "Achievement of the Week": "Achievement of the Week",
+    "Learn more about this event": "Learn more about this event",
+    "Games": "Games",
+    "Games Mastered": "Games Mastered",
+    "Games Beaten": "Games Beaten",
+    "Registered Players": "Registered Players",
+    "Achievement Unlocks": "Achievement Unlocks",
+    "Points Earned Since :date": "Points Earned Since :date",
+    "Most recent game mastered": "Most recent game mastered",
+    "Most recent game beaten": "Most recent game beaten",
+    "News": "News",
+    "Just Released": "Just Released",
+    "See more": "See more",
+    "Latest Sets in Progress": "Latest Sets in Progress",
+    "Dev": "Dev",
+    "Type": "Type",
+    "Started": "Started",
+    "New": "New",
+    "Revision": "Revision",
+    "by :authorDisplayName": "by :authorDisplayName",
+    "Currently Online": "Currently Online",
+    "Active Players": "Active Players",
+    "Viewing": "Viewing",
+    "players in-game.": "players in-game.",
+    "Couldn't find any active players.": "Couldn't find any active players.",
+    "Last updated at :timestamp": "Last updated at :timestamp",
+    "Open active players menu": "Open active players menu",
+    "players are currently online.": "players are currently online.",
+    "All-time High: :number (:date)": "All-time High: :number (:date)",
+    "Welcome!": "Welcome!",
+    "Build your profile, track your progress, and compete with friends on your favorite classic games.": "Build your profile, track your progress, and compete with friends on your favorite classic games.",
+    "We provide": "We provide",
+    "the emulators": "the emulators",
+    ", you just need": ", you just need",
+    "the games": "the games",
+    ". From Atari 2600 to PlayStation 2, and everything in between.": ". From Atari 2600 to PlayStation 2, and everything in between.",
+    "Which of these achievements do you think you can get?": "Which of these achievements do you think you can get?",
+    "Forum Recent Posts": "Forum Recent Posts",
```

Comment:
This reads weird. Suggest either "Recent Forum Posts" or "Forum - Recent Posts"

Reply:
Updated in latest.